### DISCUSSION TITLE
How to use new FancyBox 4 with jQuery?

### DISCUSSION THREAD

**Discussion by TheZoker** [fancyapps/ui#19] [Q&A]
Hi there,

Since version 4, the app seems to concentrate more on node.
But coming from a jQuery environment, I'm interested in how to include FancyBox within jQuery?

Within version 3 I was able to just use FancyBox like:
```js
$().fancybox({
    selector: '.lightbox-image',
    ...
});
```

But how can I accomblish that within version 4?

Thanks!

**Comment by fancyapps** [comment]
Hi,

The equivalent code would be -

```JS
Fancybox.bind(".lightbox-image", {
  // Your options go here
});
```

This project is written using plain JavaScript, it does not matter if you are using jQuery, React, or any other framework/tool. The source code contains some Node scripts but you are not required to use them.
